protocol/v2/types/ssvshare.go

```diff
@@ -96,6 +105,10 @@ func ComputeQuorumAndPartialQuorum(committeeSize int) (quorum uint64, partialQuo
 	return uint64(f*2 + 1), uint64(f + 1)
 }
 
+func ComputeF(committeeSize int) uint64 {
+	return uint64(committeeSize-1) / 3
+}
```

Comment:
Since we're defining a function to compute f, I think we can re-use it in `ComputeQuorumAndPartialQuorum` and `ValidCommitteeSize` to avoid knowledge duplication